### ISSUE TITLE
feedback on v1.1rc1

### ISSUE THREAD

**Issue by qralston** [msktutil/msktutil#113]
Thanks for producing a new release candidate!

`v1.1rc1` builds cleanly for me on all of Red Hat Enterprise Linux 5, 6, and 7.

Two issues, though:

1. The `v1.1rc1.tar.gz` tarball only includes `configure.in`; it doesn't include `configure`. This means that in order to build the distribution, a packager needs to have autoconf and automake installed.
2. The man page is installed mode 0755, which `rpmlint` flags. It should be mode 0644.

The man page mode is minor, but not including `configure` is an issue. There are a lot of compatibility issues across versions of autoconf/automake, so it's pretty much always preferable for the developer to package the necessary files so that builders don't have to run autoconf/automake to produce them.

(FWIW, at least for me, `autoreconf` ran cleanly for `v1.1rc1` on all of Red Hat Enterprise Linux 5, 6, and 7.)

**Comment by sckobras** [comment]
Thanks for the feedback! I've just created #114 to fix the man page permissions. As for the autoconf files in the tarball: For the pre-release, we only included the source code archives that are auto-generated by GitHub, and therefore only contain what's present in the git repository. I'll make a note that we add a proper `make dist` style tarball for the final release. (And we should probably add a `dist:` target to the `Makefile` as well.)
